src/media/js/views/homepage.js

```diff
@@ -6,6 +6,8 @@ define('views/homepage',
     var gettext = l10n.gettext;
     var appsModel = models('apps');
 
+    var $appContextmenu = document.getElementById('contextmenu').children[0];
```

Comment:
nit: I thought the prefix $ stands for a jQuery object but seems it isn't, right?